### ISSUE TITLE
Add option to use file for commit msg instead of dropdown

### ISSUE THREAD

**Issue by JasonMFry** [Microsoft/vscode#57958]
I searched existing issues but found nothing, maybe I didn't search well enough but here goes.

I would like to have a setting to choose what happens when I do `>git commit` because I don't like the current dropdown since it is not nearly as powerful as a file in VSCode. This is particularly obvious when doing `>git amend`, it smashes together both the git commit subject and body in the dropdown :(

Using a file, I can take advantage of various formatting, including linewrap.

**Comment by joaomoreno** [comment]
Have you tried using the text input box in the SCM view?

**Comment by JasonMFry** [comment]
@joaomoreno yes I have but it still lacks the power of a file in the editor, such as linewrap, font size, etc.

**Comment by JasonMFry** [comment]
@joaomoreno I read #53225 but it sounds different than what I'm asking. It seems to be requesting a larger window to type the message in. But I'm asking for the same functionality as marking VSCode as the editor in ~/.gitconfig `editor = code --wait`. Another way of stating my request is to have a command in VSCode to create a commit message respecting ~/.gitconfig
